test/execute/main.js

```diff
@@ -880,6 +880,7 @@ describe('execute', () => {
         method: 'PUT',
         credentials: 'same-origin',
         headers: {
+          accept: 'mime/silent-french-type',
```

Comment:
Not sure I understand why this change was introduced, can you please elaborate?

Reply:
Never mind I do understand now. We had to change the expectation to compensate for the change.